install/kubernetes/helm/istio/charts/tracing/templates/ingress-jaeger.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if .Values.ingress.enabled -}}
```

Comment:
Can you use the ingress gateway instead of Kubernetes Ingress? You would have to define a virtual service for the gateway and add these rules. Ingress is sort of in deprecated mode and really dont want to activate that code path

Reply:
I was wondering about that - but there are many other examples of ingress that are defined in the charts, so was thinking just bring it in line with the others first, and then they should all be updated in another PR?

Reply:
Sure..